### PR TITLE
Featurize clothes shop

### PR DESCRIPTION
**Pull request by colindamelio**
Please review my code :)
Be thoughtful, constructive, and positive.

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,84 +1,110 @@
...
+  {
+    title: 'HackerYou Hoodie',
+    price: 50.00,
+    stock: 4,
+  },
+]
 
-  // document ready function allows page to load before running any of our scripts 
-$(function(){
+storeApp.currencies = {
```
Comment (natalielaing) [inline scripts.js:47]:
```suggestion
storeApp.currencies = [
```

File: scripts.js [hunk 2]
```diff
@@ -1,84 +1,110 @@
...
+    displayName: `CAD`,
+    flag: `images/CAD-flag.png`
+  },
+  gbp: {
+    exchange: 0.76,
+    symbol: `£`,
+    displayName: `GBP`,
+    flag: `images/GBP-flag.png`
+  }
+}
```
Comment (natalielaing) [inline scripts.js:66]:
```suggestion
]
```

File: scripts.js [hunk 3]
```diff
@@ -1,84 +1,110 @@
+const storeApp = {};
```
Comment (Graingez) [inline scripts.js:2]: add notes  that say what it is each part of your code is trying to accomplish

**Review by natalielaing — COMMENTED** [review]
Overall great work, see Joey's comments for suggestions.

**Pull request closed without merging**